setup.py

```diff
@@ -41,6 +41,7 @@
     setup_requires=pytest_runner,
     tests_require=[
         'pytest>=2.8',
+        'mock>=2.0.0',
```

Comment:
you also need to add mock to tox.ini file, which is where the tox test runner is getting the test requirements.
Add a line after pytest in there.